ddtrace/contrib/django/middleware.py

```diff
@@ -56,6 +56,8 @@ def process_response(self, request, response):
             span = _get_req_span(request)
             if span:
                 span.set_tag(http.STATUS_CODE, response.status_code)
+                if response.status_code == 500:  # Some exceptions are not handled by `process_exception`
+                    span.set_traceback()
```

Comment:
Yeah I think this is not so interesting since it should contain the stack after the problem has been "resolved" by Django. Can't we catch the real exception, going deeply in the framework so that we collect valuable info for users?

Also any update for this integration requires unittest, since Django has a really good test suite for any kind of component (internals or not).